test/other/test_emit_tsd.d.ts

```diff
@@ -1,8 +1,32 @@
 // TypeScript bindings for emscripten-generated code.  Automatically generated at compile time.
+declare namespace RuntimeExports {
+    /**
+     * Given a pointer 'idx' to a null-terminated UTF8-encoded string in the given
+     * array that contains uint8 values, returns a copy of that string as a
+     * Javascript String object.
+     * heapOrArray is either a regular array, or a JavaScript typed array view.
+     * @param {number} idx
+     * @param {number=} maxBytesToRead
+     * @return {string}
+     */
```

Comment:
It seems a little redundant to include both jsdoc and typescript types here.. but I guess here is not easy way to remove them?

Reply:
Yeah, I'm not quite sure why `tsc` sometimes also includes the jsdoc. In some ways it's nice though since then you get documentation in the API.